public/css/style.css

```diff
@@ -18,6 +18,15 @@
    display: flex;
  }
 
+ /* .img-center {
+  margin-right: 30%;
+ } */
```

Comment:
Please get rid of this and every other commented out code @Daramola98

Reply:
Done @fortunee